index.bs

```diff
@@ -7151,8 +7124,9 @@ ECMAScript value type.
     1.  If <a abstract-op>Type</a>(|V|) is Boolean, then
         return the {{boolean}} value that represents the same truth value.
     1.  If <a abstract-op>Type</a>(|V|) is Number, then
-        return the result of <a href="#es-to-unrestricted-double">converting</a> |V|
-        to an {{unrestricted double}}.
+        return the result of <a href="#es-to-double">converting</a> |V|
+        to an {{double}} [=extended attribute associated with|associated with=]
```

Comment:
"a" double, not "an" double.

Also, _extended attributes_ are associated with _types_: not vice versa, as you have here. You should say "double with an [associated](https://heycam.github.io/webidl/#idl-type-extended-attribute-associated-with) [Unrestricted] extended attribute." I realize that unfortunately what you have is the norm in the spec right now, but doesn't hurt to be correct when we get a chance to.